### PR TITLE
Update libuv from 1.19.2 to 1.33.1

### PR DESCRIPTION
**Pull request by uberhacker**
Tested on all architectures.

### CODE REVIEW

File: packages/libuv.rb [hunk 1]
```diff
@@ -3,27 +3,23 @@
...
+     x86_64: '679bc3ae5d4ff33b335840fff43087fc2144619eeb678f5c047689c74840faf8',
   })
 
-  depends_on 'automake'
-  depends_on 'glibc'
-  depends_on 'libtool'
-
   def self.build
     system './autogen.sh'
     system './configure',
```
Comment (cstrouse) [inline packages/libuv.rb]: Should we be explicit and add `--libdir` and `--includedir` here?

**Comment by cstrouse** [comment]
Compiles on ARM. All but one of the tests pass FWIW.

File: packages/libuv.rb (hunk 1 above)
Comment (uberhacker) [reply packages/libuv.rb]: We already have `--libdir` and `--includedir` is usually `#{CREW_PREFIX}/include` if you set `--prefix`.

**Comment by uberhacker** [comment]
@cstrouse: Which test failed on aarch64?  Can you please paste the error output?

**Pull request merged**